src/codelabs/iterables.md

```diff
@@ -449,9 +449,9 @@ the class `User` from the previous exercise.
 Use `where` to implement two methods:
 
 * Part 1: Implement `filterUnder21`.
-  * It should return an `Iterable` of `User` with the age 21 or more.
+  * The code should return an `Iterable` of `User` with the age 21 or more.
 * Part 2: Implement `findShortNamed`.
-  * It should return an `Iterable` of `User` with the names of length 3 or less.
+  * The code should return an `Iterable` of `User` with the names that have a length of 3 or fewer.
```

Comment:
I'm going to change this to less, since we're talking math here, and they'll need to use a less-than sign.
https://www.electronicproducts.com/Software/System/Why_we_use_a_less_than_sign_and_not_a_fewer_than_sign.aspx

Reply:
OK. Didn’t realize that a less than sign needed to be used.

Reply:
It's a weird math thing. It took me a while to figure out the reason why "fewer" seemed wrong, when it was right in the English sense.